examples/README.md

```diff
@@ -12,14 +12,19 @@ Currently, the following scenarios are included:
 
 ### Lodging
 
-#### Shopping
+- [`ListingsQuickStartScenario.java`](src/main/java/com/expediagroup/sdk/xap/examples/scenarios/lodging/ListingsQuickStartScenario.java):
```

Comment:
Rename to ListingsSearchingByLocationKeyword.java?

Reply:
What about Quotes and AvailCalendars? They are still named as QuickStart.

Reply:
The Quotes case, we need to update it to get property list through SDP first and then perform the search. However I think we can keep using the existing names for them.

Reply:
Discussed on daily meeting. Quick start is the simplest case to successfully make a call to the service. This scenario can remain as it is.